src/domains.js

```diff
@@ -8,10 +8,10 @@ import {
 } from "@krakenjs/cross-domain-utils/src";
 
 import {
-  getProtocol,
-  getStageHost,
   getPayPalDomain,
   getPayPalAPIDomain,
+  getStageHost,
```

Comment:
totally a nit but if this order was changed, why not alphabetical?

Reply:
Fixed!